hail/python/hail/docs/conf.py

```diff
@@ -224,7 +224,14 @@
 #
 www_root = os.path.dirname(os.path.abspath(__file__))
 _i = 0
-while _i < 50 and not os.path.isdir(os.path.join(www_root, 'www')):
+while _i < 50:
+    if(os.path.isdir(os.path.join(www_root, 'www'))):
+        break
+
+    if(os.path.isdir(os.path.join(www_root, 'site'))):
```

Comment:
since build depends on www we need 2 paths, or a much larger commit in which we rename site to www

Reply:
These files aren't actually referenced in the docs. The docs refer directly to `/navbar.css`. It seems like we should just remove lines 225 through 228 and reduce `html_extra_path` to just `misc/`. If we want the navbar to be present when viewing the docs locally, we should create a `hail/Makefile` target that sets up the environment correctly or a `site/Makefile` target that runs the whole site locally.

IMO, when I look at the docs locally it's just to check that I got my RST syntax right, I don't think the nabber is so important to that.

Reply:
I did try to get a local site working correctly but it was kind of annoying to parameterize nginx's TLS setting and I found `make -C site deploy NAMESPACE=dking` pretty fast anyway

Reply:
But I'm definitely here for a good local site preview.

Reply:
Can we use python -m http.server? I don't think nginx is needed for site.